e2e_tests/src/e2e_indexer/validate_accounting.py

```diff
@@ -665,6 +750,14 @@ def compare(indexer_account, i2a_checker, algod, indexer, indexer_headers):
         return
     algod_account = algod.account_info(niceaddr)
     if algod_account["round"] != indexer_account["round"]:
+        logger.warning(
+            f'round mismatch: {algod_account["round"]=} VS {indexer_account["round"]=}: {niceaddr=}'
+        )
+        # TODO: Don't merge with the following comments
+        # return
+        # this no longer is guaranteed to work because not all accounts are rewindable
+        # EG:
+        # '{"message":"failed while searching for account: error while rewinding account: ...[55,13]: rewinding past txn type appl is not currently supported"}
```

Comment:
Strange behavior for failing tests - failing while trying to rewind an account with an application transaction. Seems like this issue would be present regardless of the difference between `rel-nightly` and `stable`.

Reply:
I remember something similar happening when I was working on these tests before. I think it was just from a mismatch between rounds...i.e. we're not successfully importing in the way we should be, or there was a failure happening that was not evident without looking into some specific logs. Sorry if that's not super helpful.

Reply:
(not resolving to show the original comment that I'm about to remove)